zrcs/zrc-2.md

```diff
@@ -0,0 +1,412 @@
+transition RevokeOperator(operator: ByStr20)
+```
+
+|        | Name       | Type      | Description                      |
+| ------ | ---------- | --------- | -------------------------------- |
+| @param | `operator` | `ByStr20` | Address to be unset as operator. |
+
+|           | Name                       | Description                             | Event Parameters                                                                                                                                                                                                               |
+| --------- | -------------------------- | --------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| eventName | `RevokeOperatorSuccess` | Revoking is successful.     | `operator`: `ByStr20`. |
+| eventName | `Error`                    | Revoking is not successful. | - emit `CodeNotAuthorised` if the transition is called by a user who is not the token holder.                                                                                                      |
+
+
+#### 11. IsOperatorFor
+
+```ocaml
+(* @dev: Returns true if an address is an operator of tokenOwner. All addresses are their own operator. *)
+(* @param operator:    Address of a potential operator.                                                 *)
+(* @param tokenOwner:  Address of a token holder.                                                       *)
+transition IsOperatorFor(operator: ByStr20, tokenOwner: ByStr20)
+```
+
+|        | Name          | Type      | Description                          |
+| ------ | ------------- | --------- | ------------------------------------ |
+| @param | `operator`    | `ByStr20` | Address of a potential operator.     |
+| @param | `tokenOwner`  | `ByStr20` | Address of a token ownwer.           |
+
+|           | Name                       | Description                             | Event Parameters                                                                                                                                                                                                               |
+| --------- | -------------------------- | --------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| eventName | `IsOperatorForSuccess` | Checking operator is successful.     | `operator`: `ByStr20`, and `tokenOwner`: `ByStr20`. |
+| eventName | `Error`                    | Checking operator is not successful. | TBA.                                                                                                      |
```

Comment:
I don't think there should be any error here at all. It is just a `True` or `False` result (event).

Reply:
fixed.